src/app/Scenes/Home/useHomeModules.ts

```diff
@@ -58,14 +57,15 @@ export const useHomeModules = (props: HomeProps) => {
         type: "artwork",
       },
       {
-        contextModule: ContextModule.auctionLotsEndingSoonRail,
-        data: props.meAbove,
-        isEmpty: !props.meAbove?.lotsByFollowedArtistsConnectionCount?.edges?.length,
-        key: "lotsByFollowedArtistsRail",
-        prefetchUrl: "/auctions/lots-for-you-ending-soon",
-        prefetchVariables: lotsByArtistsYouFollowDefaultVariables(),
-        title: "Auction Lots for You Ending Soon",
-        type: "lotsByFollowedArtists",
+        contextModule: ContextModule.lotsForYouRail,
+        data: props.recommendedAuctions,
+        isEmpty: isEmpty(props.recommendedAuctions),
+        key: "recommendedAuctionsRail",
+        //TODO: update prefetch URL
+        // prefetchUrl: "/auctions",
+        // prefetchVariables: lotsByArtistsYouFollowDefaultVariables(),
```

Comment:
If we keep the link, we can also keep the old prefetching logic:

```suggestion
        prefetchUrl: "/auctions/lots-for-you-ending-soon",
        prefetchVariables: lotsByArtistsYouFollowDefaultVariables(),
```